package.json

```diff
@@ -0,0 +1,104 @@
+{
+  "name": "network-dashboard",
+  "description": "Aragon Network Dashboard",
+  "version": "0.1.0",
+  "private": true,
+  "license": "AGPL-3.0-or-later",
+  "author": "Aragon Association <legal@aragon.org>",
+  "repository": {
+    "type": "git",
+    "url": "git+ssh://git@github.com/aragon/network-dashboard.git"
+  },
+  "bugs": {
+    "url": "https://github.com/aragon/network-dashboard/issues"
+  },
+  "homepage": "https://github.com/aragon/network-dashboard#readme",
+  "keywords": [
+    "aragon",
+    "network",
+    "ethereum",
+    "dapp",
+    "blockchain",
+    "startups"
+  ],
+  "engines": {
+    "node": ">= 12.0.0"
+  },
+  "bundlewatch": {
+    "files": [
+      {
+        "path": "public/*.js",
+        "maxSize": "1600kb",
```

Comment:
Perhaps we can set out a smaller target for this app to begin with :). Hopefully it'll be much, much lighter than the client!

Reply:
Fair, I've arbitrarily halved it.